pegasus/sites.v3/hourofcode.com/views/signup_form.haml

```diff
@@ -106,20 +106,20 @@
             %input#hoc-match-volunteer-flag{name: "match_volunteer_flag_b", type: "checkbox", value: "1"}/
             = hoc_s(:signup_match_volunteer)
 
-    .form-group.continue-btn{style: "display: none;"}
-      %div
-        %button.submit-btn#continue{type: "button"}
-          = hoc_s(:signup_submit_and_continue_label)
-
     #submit
       %input{name:'hoc_country_s', type:'hidden', value:@country}
       %input{name:'hoc_company_s', type:'hidden', value:@company}
       %p.footnote
         = hoc_s(:signup_multiple_event_warning)
         %strong= hoc_s(:signup_registration_deadline)
-      .form-group.submit-btn
+      .form-group.submit-btn#submit-button
```

Comment:
I'm confused by the inconsistency between the two submit buttons. Should the id and class be applied to the `.form-group` or the `%button` itself?